app/src/main/res/layout/view_pending_attachment.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.thoughtcrime.securesms.conversation.v2.messages.PendingAttachmentView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="horizontal"
+    android:paddingHorizontal="12dp"
+    android:paddingVertical="@dimen/small_spacing"
+    android:gravity="center">
```

Comment:
We need to discuss this with Becca and Lucy to decide what the expectation is around this feature since it isn't fully implemented (for example the icon is never changing currently)